src/api/ledger/parse/trustline.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const assert = require('assert');
+const utils = require('./utils');
+const flags = utils.core.Transaction.flags.TrustSet;
+
+function parseTrustline(tx) {
```

Comment:
I probably shouldn't have been "reviewing" this, as I was never familiar
with ripple-rest / ripple-api, it's just no one had yet.

I'll stay away from these in the future.